test/verify/check-dashboard

```diff
@@ -94,7 +94,7 @@ class TestBasicDashboard(MachineCase, DashBoardHelpers):
 
         # Start second browser and check that it is in sync
         b2 = self.new_browser()
-        b2.default_user = "root"
+        b2.default_user = "admin"
```

Comment:
This could also be split out and landed  separately. (If it helps, then you don't need to run the entire set of tests for this PR)